.travis.yml

```diff
@@ -28,3 +28,5 @@ install:
 script:
   - python -m unittest discover tests -v
   - pytest --cov tests
+  - bash <(curl -s https://codecov.io/bash)
+  
```

Comment:
```suggestion

```

Reply:
@devkapilbansal don't you suggest terminating with a new line ?

Reply:
I do but that line contains spaces. And blank line shouldn't contain anything

Reply:
Done 😉